Lecture3/Task1/product/ProductFactoryDirector.java

```diff
@@ -0,0 +1,44 @@
+package product;
+
+import builder.AcousticGuitarBuilder;
+import builder.ElectricGuitarBuilder;
+import builder.Guitar;
+import builder.GuitarBuilderDirector;
+import exceptions.FailedOrderException;
+import utils.RandomUtils;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class ProductFactoryDirector {
+
+	public static List<Guitar> createGuitars() throws FailedOrderException {
+		List<Guitar> guitars = new ArrayList<>();
+		GuitarBuilderDirector builderDirector = new GuitarBuilderDirector();
+		if (guitars.size() == 0) {
```

Comment:
as you have created new ArrayList in 16 line this condition (if (guitars.size() == 0)) - will be always true
so remove it

Reply:
done